src/main/java/data/ResponseBuilder.java

```diff
@@ -0,0 +1,26 @@
+package data;
+
+import util.UserAuthUtil;
+
+/** Class to build Login/Logout response objects in JSON format */
+public class ResponseBuilder {
```

Comment:
I would like to see tests for this (okay to add after the demo).

Reply:
#25